### PR TITLE
Cypress/integration/captures filter tests

### PR DESCRIPTION
**Pull request by PtrHtn**
## Description
Fixes  Greenstand/treetracker-admin-client#303
Tests for Captures Filter

## Breaking change
**Does this PR introduce a breaking change?**
No breaking changes

### CODE REVIEW

File: cypress/fixtures/treesFilter_Mock.json [hunk 1]
```diff
@@ -0,0 +1,427 @@
+[
+  {
+    "id": 666666,
+    "timeCreated": "+052341-09-10T13:56:40.000Z",
+    "planterId": 5290,
+    "active": true,
+    "planterIdentifier": "arunbakt@live.com",
```
Comment (nmcharlton) [inline cypress/fixtures/treesFilter_Mock.json:7]: I'd suggest using made up email addresses or phone numbers here.

**Review by nmcharlton — APPROVED** [review]
Looking good

File: cypress/fixtures/treesFilter_Mock.json (hunk 1 above)
Comment (PtrHtn) [reply cypress/fixtures/treesFilter_Mock.json]: The email addresses have been modified.

File: cypress/support/pages/GrowersPage.js.orig [hunk 2]
```diff
@@ -0,0 +1,49 @@
+class GrowersPage {
```
Comment (nmcharlton) [inline cypress/support/pages/GrowersPage.js.orig:1]: This file looks like it shouldn't have been committed.

File: cypress/support/pages/HomePage.js.orig [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class HomePage {
```
Comment (nmcharlton) [inline cypress/support/pages/HomePage.js.orig:1]: Are these auto-generated? If so, we should add `*.orig` to .gitignore.

File: cypress/support/pages/LoginPage.js.orig [hunk 4]
```diff
@@ -0,0 +1,77 @@
+const credentials = require('../../fixtures/login.json');
```
Comment (nmcharlton) [inline cypress/support/pages/LoginPage.js.orig:1]: Another `.orig` file that shouldn't be here.

**Review by nmcharlton — CHANGES_REQUESTED** [review]
Some rogue files to remove before merge.

File: cypress/support/pages/HomePage.js.orig (hunk 3 above)
Comment (PtrHtn) [reply cypress/support/pages/HomePage.js.orig]: *.orig was added to .gitignore

**Review by nmcharlton — APPROVED** [review]

**Pull request merged**